### PR TITLE
Update for SpongeAPI 7.2

### PR DESCRIPTION
**Pull request by ImMorpheus**
The [cookbook](https://github.com/SpongePowered/Cookbook) repo has been archived and the docs is still pointing to a few examples from it. What's the plan ?

### CODE REVIEW

File: source/contributing/spongedocs.rst [hunk 1]
```diff
@@ -49,7 +49,7 @@ writing Sponge Documentation. This list may get added to (or bent out of shape)
 
 1. Headings Should Be Written in Title Case (<- example) [unless #8 applies].
 2. Page headings should be meaningful (the heading appears as a link).
-3. Program code should be contained in `inline literals <http://docutils.sourceforge.net/docs/ref/rst/roles.html#literal>`__
+3. Program code should be contained in `inline literals <https://docutils.sourceforge.net/docs/ref/rst/roles.html#literal>`__
```
Comment (ST-DDT) [inline source/contributing/spongedocs.rst]: The https link does not work for me.
```suggestion
3. Program code should be contained in `inline literals <https://docutils.sourceforge.io/docs/ref/rst/roles.html#literal>`__
```

File: source/plugin/injection.rst [hunk 2]
```diff
@@ -27,13 +33,33 @@ Temporary List of Injected Types
...
+:javadoc:`SynchronousExecutor` (annotation on SpongeExecutorService)
+  Used to inject the plugin's specific SynchronousExecutor
+
+:javadoc:`ChannelId` (annotation on ChannelBinding.IndexedMessageChannel or ChannelBinding.RawDataChannel)
+  Used to inject a ``ChannelBinding`` with the given channel id
+
+:javadoc:`Asset`
+  Must be annotated with ``@AssetId``.
+
+:javadoc:`SpongeExecutorService`
```
Comment (ST-DDT) [inline source/plugin/injection.rst:55]: Missing import

**Review by ST-DDT — CHANGES_REQUESTED** [review]
Thanks for checking the docs.
I only found two small issues and then it should be ready for merge

**Review by ST-DDT — APPROVED** [review]
LGTM

**Review by Inscrutable — APPROVED** [review]

**Pull request merged**